kube/resources/connector-builder-server.yaml

```diff
@@ -1,3 +1,15 @@
+apiVersion: v1
```

Comment:
this file is not imported in the kube deploys (https://github.com/airbytehq/airbyte/blob/master/kube/resources/kustomization.yaml#L7)